seed_data/seedData.js

```diff
@@ -12,37 +12,49 @@ var freelancers = {
       "_id"         : "5837133bcb98f316ac473853",
       "firstName"   : "John",
       "familyName"  : "Smith",
+      "title"       : "President Candidate",
```

Comment:
Not really a "job" that a freelancer can offer. What would you hire a `President Candidate` for (except making America great again)?